generate_workspace/src/main/java/com/google/devtools/build/workspace/maven/Rule.java

```diff
@@ -16,18 +16,18 @@
 
 import com.google.common.base.Preconditions;
 import com.google.common.collect.Sets;
-
-import java.lang.invoke.MethodHandles;
-import java.util.logging.Logger;
-import javax.annotation.Nullable;
 import org.eclipse.aether.artifact.Artifact;
 
+import javax.annotation.Nullable;
+import java.lang.invoke.MethodHandles;
 import java.util.Objects;
 import java.util.Set;
+import java.util.logging.Logger;
 
 /**
  * A struct representing the fields of maven_jar to be written to the WORKSPACE file.
  */
+//TODO(petros): Kill this after refactoring resolvers.
```

Comment:
This is the direction I am moving with [the next few commits](https://github.com/petroseskinder/migration-tooling/commit/14880cc251a907af40c8a696f4ac15ed7a4368d3#diff-376ae4a942b0ed1920d7f8b28cf11751)